src/signals/incident-management/components/MapInput/index.js

```diff
@@ -1,28 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import MapInteractive from 'components/MapInteractive';
+import MapContext from 'containers/MapContext';
+import MapInputComponent from 'components/MapInput';
+import MAP_OPTIONS from 'shared/services/configuration/map-options';
+import { formatMapLocation } from 'shared/services/map-location';
 
 import Label from 'components/Label';
 
 export const MapInput = props => {
   const { name, display, onQueryResult } = props;
-  const render = ({ handler }) => (
-    <div className="map-input">
-      <div className="mode_input map rij_verplicht">
-        <Label htmlFor={`form${name}`}>{display}</Label>
+  const render = ({ handler }) => {
+    const value = formatMapLocation(handler().value || {});
+    console.log(value, handler().value);
+    const { lat, lng } = value?.location || {};
+    const mapOptions = {
+      ...MAP_OPTIONS,
+      center: (lat && lng) ? [lat, lng] : [...MAP_OPTIONS.center],
+    };
 
-        <div className="map-input__control invoer">
-          <MapInteractive location={handler().value} onQueryResult={onQueryResult} />
-        </div>
+    const onLocationChange = location => {
```

Comment:
useCallback`

Reply:
Can't, is not possible to nest hooks.

Reply:
Ah yes, because it's that stupid `react-reactive-form` render method 🙄